src/mappings/Shelf.ts

```diff
@@ -25,17 +25,17 @@ export function handleShelfIssue(call: IssueCall): void {
   let loanId = loanIdFromPoolIdAndIndex(poolId, loanIndex)
 
   let pool = Pool.load(poolId)
-  let poolChanged = false
 
   if (!pool.loans.includes(poolId)) {
```

Comment:
I think this should be:
```suggestion
  if (!pool.loans.includes(loanId)) {
```

Reply:
Happy to merge it into this branch as a next step 👍

Reply:
I think you can merge this in now right?

Reply:
Done in 
a5ff0e3